package.json

```diff
@@ -42,8 +43,12 @@
     "@babel/eslint-parser": "^7.16.5",
     "@babel/plugin-transform-runtime": "^7.16.5",
     "@babel/preset-env": "^7.16.5",
+    "babel-jest": "^29.5.0",
     "eslint": "^8.5.0",
-    "eslint-config-airbnb": "^19.0.2"
+    "eslint-config-airbnb": "^19.0.2",
+    "jest": "^29.5.0",
+    "jest-cli": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0"
   },
   "dependencies": {
     "@babel/runtime": "^7.16.5"
```

Comment:
I think we can move this to devDependencies can't we? It's used to build the files but it is not a dependency for the framework.

Reply:
Oof! We're going on 4 years of installing that on our projects for no good reason 😬 (07/23/2019)

Reply:
Yeah, I think that was overlooked! Glad we are giving this some TLC.